heidegger_index/templates/lemma_detail.html

```diff
@@ -13,6 +13,41 @@ <h1 class="clear-right pb-8 text-5xl font-semibold dark:text-white{% if lemma.ty
 </div>
 {% endif %}
 
+{% if lemma.type == "w" and lemma.urn and work_full_text%}
+<div class="{{ source_text }}">
+  <p class="indent-4">
+    {{ work_full_text }}
+  </p>
+  <p class="text-sm text-right">
```

Comment:
Ik zou deze tekst iets groter maken.

Reply:
Bedoel je de tekst van de link of de complete 'work_full_text'?

Reply:
Die link bedoelde ik ja, `text-lg` ipv `text-sm` lijkt me klein genoeg. (Die groottes lijken overigens wel een beetje uit verhouding, `text-lg` is nou niet echt large te noemen..)

Reply:
Bij deze:
![Screenshot 2022-09-03 at 10 35 37](https://user-images.githubusercontent.com/77152231/188262982-d428181f-42dc-46f6-9b68-8375937522fd.png)